tests/Operation/ListDatabasesFunctionalTest.php

```diff
@@ -52,6 +52,25 @@ public function testFilterOption()
         }
     }
 
+    public function testOffsetGet()
```

Comment:
These would be more suited as unit tests within the existing model test classes (DatabaseInfoTest and CollectionInfoTest). The following IndexInfoTest should work as prior art:

 * `testImplementsArrayAccess()`
 * `testOffsetSetCannotBeCalled()`
 * `testOffsetUnsetCannotBeCalled()`

Reply:
Unlike the CollectionInfo functional test (for [SERVER-26513](https://jira.mongodb.org/browse/SERVER-26513)), I don't think we need a functional test for extra fields in DatabaseInfo.